docs/general/manage-tokens/how-to-transfer-rose-erc20.md

```diff
@@ -0,0 +1,269 @@
+# How to Bridge Assets to Oasis Sapphire
+
+This guide shows how to **bridge assets (ETH, USDC, USDT, etc.)** from networks
+like Ethereum, BNB Chain, or Polygon to **Oasis Sapphire** - an EVM-compatible
+ParaTime with confidential smart contracts.
+
+**Main Bridge:** [**Celer cBridge**][cbridge] - a decentralized, non-custodial
+bridge supporting fast transfers from multiple networks.
+
+This guide covers:
+
+- Bridging assets via cBridge
+- Transferring ROSE to Sapphire
+- Important warnings and FAQs
+
+**Prerequisites:**
+
+- Web3 wallet (e.g. MetaMask) with Oasis Sapphire network added
+- Gas fees on source chain (ETH, BNB, etc.)
+
+:::tip
+Have some **ROSE on Sapphire** for gas fees to use your bridged tokens.
+See [Transferring ROSE](#transferring-rose-to-sapphire-from-exchanges-or-oasis-wallet) below.
+:::
```

Comment:
I'm not sure if I understood this admonition correctly. If a user needs native ROSE, they should skip the first section?

Anyway, I would point them to https://docs.oasis.io/general/manage-tokens/#obtaining-rose

```suggestion
:::info Need some ROSE on Sapphire for gas fees to use your bridged tokens?

See [Transferring ROSE](#transferring-rose-to-sapphire-from-exchanges-or-oasis-wallet) below.

:::
```